sdk/identity/identity/CHANGELOG.md

```diff
@@ -1,15 +1,13 @@
 # Release History
 
-## 3.1.0 (Unreleased)
-
-### Features Added
-
-### Breaking Changes
-
-### Bugs Fixed
+## 3.1.0 (2022-11-08)
 
 ### Other Changes
 
+- Includes bug fix for `additionallyAllowedTenants` to pass through to MSAL Flow from 3.0.1 release
```

Comment:
why does this need to be repeated? are there things dropped from v3.0.1 in this release?